app/next-client-app/components/HandleArchive.tsx

```diff
@@ -21,7 +21,7 @@ export const HandleArchive = async ({
       response = await archiveDataSets(id, !hidden);
       break;
     case "scanreports":
-      response = await updateScanReport(id, "hidden", !hidden);
+      response = await updateScanReport(id, { hidden: !hidden });
```

Comment:
I don't like how this reads. If `hidden` is `true`, meaning that something should be hidden, this code implies that in fact it should NOT be hidden.

Reply:
so, when users click Archive/unarchive, the function will do its job which is receiving the current value of "hidden" (false/true) and sending to the backend the opposite value (true/false). in terms of logic, this code is doing well.
You don't like the naming or something else?

Reply:
I see what you mean now. Basically it kind of felt a bit odd on reading. But if it's doing what you expect then fine